src/style.css

```diff
@@ -1 +1,115 @@
-/* empty source */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3 ;
+}
+
+.container__image {
+  height: 134px;
+  margin: 0 auto 40px;
+  width: 160px;
+}
+
+.container__text {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.container__product-code {
+  margin-bottom: 16px;
+}
+
+.container__stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: center;
+
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  margin-bottom: 24px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.reviews {
+  color: #060b35;
+}
+
+.container__item-price {
+  display: flex;
+  justify-content: space-between;
+
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.container__price {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+  margin-bottom: 16px;
```

Comment:
is is actually a `container`?
it just has a text in it - but the real container is `.container__item-price` - so apply margin there, not here
<img width="658" alt="margin" src="https://user-images.githubusercontent.com/39308717/200532144-668c2052-7dfb-4c8f-bd01-1ece37315611.png">